server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/task/InfinispanIntegrationMetadataFileFinder.java

```diff
@@ -0,0 +1,25 @@
+package org.jboss.as.clustering.infinispan.task;
```

Comment:
Without it the component metadata from *.dat file is not imported (I've just checked it)